### PR TITLE
[WHO-186] ui layout v 0.1

### PR DESCRIPTION
**Pull request by myiremark**
## What does this PR accomplish?
Brings UI up to figma specifications for Monday process review

- ShareTheApp, SendFeedback, and About left as placeholders per other issues
- Diverges from UI specification to allow for content change/proof of concept internationalization without changing layout mechanism
- Removes unused prototyping functionality
- Makes title bar uniform across various pages

## Work needed after merge
- *NEEDS STYLING* (https://github.com/WorldHealthOrganization/app/issues/198)
- Requires Content Flow for Protect Yourself (https://github.com/WorldHealthOrganization/app/issues/196)
- Requires Content Flow for Check Your Health (https://github.com/WorldHealthOrganization/app/issues/197)

[List all issue numbers affected and closed by this PR.]

## Did you add any dependencies?
None

## How did you test the change?
Clicked through all routes, screenshots included

### CODE REVIEW

**Comment by myiremark** [comment]
04_dynamic_content
03_menu
02_splash_carousel
01_splash_info
00_splash_title

File: client/ionic/src/App.tsx [hunk 1]
```diff
@@ -38,21 +38,17 @@ const App: React.FC = () => (
     <IonReactRouter>
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
-        <Route
-          path="/coping-everyone"
-          component={CopingEveryone}
-          exact={true}
-        />
-        <Route path="/coping-parents" component={CopingParents} exact={true} />
+        <Route path="/splash-title" component={SplashTitle} exact={true} />
```
Comment (hspinks) [inline client/ionic/src/App.tsx]: @viviancromwell can you clarify the intention of the "Getting Started - first time user" flow? How does the user navigate through that?

@myiremark general comment that I'm leaving here is that I'm guessing user press on the card isn't how you navigate through, but will let Vivian speak to it

File: client/ionic/src/pages/About.tsx [hunk 2]
```diff
@@ -1,82 +1,13 @@
...
-                /** TODO: Handle errors of unsupported screen types correctly. */
-              }
-            })}
-          </IonSlides>
-        )}
-        <IonButton className="pt2" href="/menu" shape="round">
-          Learn More
-        </IonButton>
-      </IonContent>
+      <TopNav />
```
Comment (hspinks) [inline client/ionic/src/pages/About.tsx]: what is this page now?

File: client/ionic/src/pages/CheckYourHealth.tsx [hunk 3]
```diff
@@ -11,13 +10,12 @@ import {
...
 import 'tachyons';
-import { getUserContext } from '../content/userContext';
 import useDynamicFlow from '../hooks/useDynamicFlow';
 
-const CopingParents: React.FC = () => {
+const CheckYourHealth: React.FC = () => {
   // TODO: Refactor this out to separate Flow components. Use a dictionary
   // of screen archetypes.
-  const flow = useDynamicFlow('copingParents');
+  const flow = useDynamicFlow('check');
```
Comment (hspinks) [inline client/ionic/src/pages/CheckYourHealth.tsx]: I'm worried about future collisions on the name `check` for the flow, let's choose something more specific to this page

File: client/ionic/src/pages/About.tsx (hunk 2 above)
Comment (myiremark) [reply client/ionic/src/pages/About.tsx]: I wasn't certain of the content for the bottom 3 pages of the menu so i just left them blank for the time being.

File: client/ionic/src/pages/CheckYourHealth.tsx (hunk 3 above)
Comment (advayDev1) [reply client/ionic/src/pages/CheckYourHealth.tsx]: we should use a standard atm of react name converted to kebab case is flow.

Comment (advayDev1) [inline client/ionic/src/pages/CheckYourHealth.tsx]:
```suggestion
  const flow = useDynamicFlow('check-your-health');
```

File: client/ionic/src/pages/SplashCarousel.tsx [hunk 4]
```diff
@@ -1,48 +1,57 @@
...
-import TopNav from '../components/TopNav';
 import 'tachyons';
-import { getUserContext } from '../content/userContext';
 import useDynamicFlow from '../hooks/useDynamicFlow';
+import TopNav from '../components/TopNav';
+
+import { S } from '../i18n/S';
 
-const CopingEveryone: React.FC = () => {
+const Splash: React.FC = () => {
```
Comment (advayDev1) [inline client/ionic/src/pages/SplashCarousel.tsx]:
```suggestion
const SplashCarousel: React.FC = () => {
```

let's keep the export name = file names for components using default export

File: content/flows/v1/check.ww.en.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+type: LinearCarousel
```
Comment (advayDev1) [inline content/flows/v1/check.ww.en.yaml]: rename file to check-your-health.ww.en.yaml

File: client/ionic/src/pages/CheckYourHealth.tsx (hunk 3 above)
Comment (myiremark) [reply client/ionic/src/pages/CheckYourHealth.tsx]: done in upgraded version

File: client/ionic/src/pages/SplashCarousel.tsx (hunk 4 above)
Comment (myiremark) [reply client/ionic/src/pages/SplashCarousel.tsx]: done

File: content/flows/v1/check.ww.en.yaml (hunk 5 above)
Comment (myiremark) [reply content/flows/v1/check.ww.en.yaml]: done

File: client/ionic/src/pages/CheckYourHealth.tsx (hunk 3 above)
Comment (myiremark) [reply client/ionic/src/pages/CheckYourHealth.tsx]: done

**Comment by advayDev1** [comment]
Because @hspinks  and I have Approved, and because the Netlify dashboard says:
> All linked resources are secure
> Congratulations! No insecure mixed content found in your files.
 I am authorizing bypassing the 20+ minute bugged out Netlify Mixed Content CI check and merging anyway.

**Pull request merged**